tests/unit/test_alteration.py

```diff
@@ -136,3 +138,70 @@ def test_validate_invalid_share_structure_alteration():
     print(errors)
 
     assert not is_valid
+
+
+@pytest.mark.parametrize('invalid_file_number', [
+    '1234',
+    '123456789012345678901',
+    True,
+    ['123456']
+])
+def test_validate_invalid_court_order_file_numbers(invalid_file_number):
+    """Assert not valid court order file numbers."""
+    alteration_json = copy.deepcopy(ALTERATION)
+    alteration_json['courtOrder']['fileNumber'] = invalid_file_number
+
+    is_valid, errors = validate(alteration_json, 'alteration')
+
+    if errors:
+        for err in errors:
+            print(err.message)
+    print(errors)
+
+    assert not is_valid
+
+
+def test_validate_invalid_court_order_order_date_format():
+    """Assert not valid court order order date format."""
+    alteration_json = copy.deepcopy(ALTERATION)
+    alteration_json['courtOrder']['orderDate'] = '2021-01-30T09:56:01'
+
+    is_valid, errors = validate(alteration_json, 'alteration')
+
+    if errors:
+        for err in errors:
+            print(err.message)
+    print(errors)
+
+    assert not is_valid
+
+
+@pytest.mark.parametrize('invalid_effect_of_order', [
```

Comment:
could all 3 of these test be combined into a parameterized set of tests.
Also, if you're trying to fuzz the inputs, you can use the _hypothesis_ package used elsewhere.